ext/fg/js/popup.js

```diff
@@ -117,8 +117,8 @@ class Popup {
     }
 
     async containsPoint(x, y) {
-        for (let popup = this; popup !== null && popup.isVisibleSync(); popup = popup._child) {
-            const rect = popup._container.getBoundingClientRect();
+        for (let popup = this; popup !== null && popup.isVisibleSync(); popup = this._child) {
+            const rect = this._container.getBoundingClientRect();
```

Comment:
This actually happened because I did a find and replace of "Popup." with "this."; guess the case sensitivity check was off.
06caa29666a3dc3b9726b77903a77568c45fedfe

Reply:
Ah, makes sense :smile: